src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -303,5 +305,27 @@ public void nerfFishing(PlayerFishEvent event) {
         caught.setItemStack(new ItemStack(Material.RAW_FISH));
       }
     }
+
+
+  }
+
+  @EventHandler(priority = EventPriority.MONITOR)
+  public void dropItemsOnQuit(PlayerParticipationStopEvent event) {
+    MatchPlayer quitter = event.getPlayer();
+    Player player = Bukkit.getPlayer(quitter.getId());
+
+      for (ItemStack i : quitter.getInventory().getContents()) { //Drop inventory
+        if (i != null) {
+          quitter.getWorld().dropItemNaturally(player.getLocation(), i);
+        }
+      }
+
+      ItemStack[] playerArmor = quitter.getInventory().getArmorContents(); //Drop Armor
+      for (int i = 0; i < 4 ; i++) {
+        if(playerArmor[i].getType() == Material.AIR){ //Skips empty armor slots
+          continue;
+        }
+        quitter.getWorld().dropItemNaturally(player.getLocation(), playerArmor[i]);
+      }
```

Comment:
Are you sure about this? I think `getContents()` already returns the whole inventory with the armor.

Reply:
No, just tested it again to be sure. When i remove this part of the code it only drops items from the Inventory.